tools/clang/unittests/HLSL/ExtensionTest.cpp

```diff
@@ -183,14 +189,15 @@ Intrinsic Intrinsics[] = {
   // Make this intrinsic have the same opcode as an hlsl intrinsic with an unsigned
   // counterpart for testing purposes.
   {L"test_unsigned","test_unsigned",   "n", { static_cast<unsigned>(hlsl::IntrinsicOp::IOP_min), false, true, false, -1, countof(TestUnsigned), TestUnsigned}},
-  {L"wave_proc",    DEFAULT_NAME,      "r", { 16, false, true, true, -1, countof(WaveProcArgs), WaveProcArgs }},
-  {L"test_o_1",     "test_o_1.$o:1",   "r", { 18, false, true, true, -1, countof(TestOverloadArgs), TestOverloadArgs }},
-  {L"test_o_2",     "test_o_2.$o:2",   "r", { 19, false, true, true, -1, countof(TestOverloadArgs), TestOverloadArgs }},
-  {L"test_o_3",     "test_o_3.$o:3",   "r", { 20, false, true, true, -1, countof(TestOverloadArgs), TestOverloadArgs }},
+  {L"wave_proc",    DEFAULT_NAME,      "r", { 16, false, true, true, -1,  countof(WaveProcArgs), WaveProcArgs }},
```

Comment:
Do we need to change these lines? Having trouble seeing the actual diff.

Reply:
This is whitespace to make things line up nicer. I can undo if it's distracting

Reply:
Fine with me as long as it's serving a purpose. Was just a bit hard to tell from the diff if it was intentional.